scripts/bosh-lite.sh

```diff
@@ -0,0 +1,367 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+SCRIPT_NAME="$0"
+PROJECT_DIR="$(cd "$(dirname "${SCRIPT_NAME}")/.."; pwd)"
+PROJECT_DIR_PARENT="$(cd "${PROJECT_DIR}/.."; pwd)"
+
+# Set to false to disable autohalt
+BOSH_LITE_AUTO_HALT=${BOSH_LITE_AUTO_HALT:-true}
+
+# Defaults
+DEFAULT_BOSH_LITE_INSTANCE_TYPE=m3.xlarge
+DEFAULT_BOSH_LITE_CODEBASE_PATH="${PROJECT_DIR_PARENT}/bosh-lite"
+DEFAULT_BOSH_LITE_REGION="eu-west-1"
+DEFAULT_BOSH_LITE_SUBNET_TAG_NAME="bosh-lite-subnet-0"
+DEFAULT_BOSH_LITE_SECURITY_GROUP_TAG_NAME="bosh-lite-office-access"
+
+export BOSH_LITE_INSTANCE_TYPE="${BOSH_LITE_INSTANCE_TYPE:-${DEFAULT_BOSH_LITE_INSTANCE_TYPE}}"
+export BOSH_LITE_CODEBASE_PATH="${BOSH_LITE_CODEBASE_PATH:-${DEFAULT_BOSH_LITE_CODEBASE_PATH}}"
+export BOSH_LITE_REGION="${BOSH_LITE_REGION:-${DEFAULT_BOSH_LITE_REGION}}"
+
+export BOSH_LITE_SUBNET_TAG_NAME="${BOSH_LITE_SUBNET_TAG_NAME:-${DEFAULT_BOSH_LITE_SUBNET_TAG_NAME}}"
+export BOSH_LITE_SECURITY_GROUP_TAG_NAME="${BOSH_LITE_SECURITY_GROUP_TAG_NAME:-${DEFAULT_BOSH_LITE_SECURITY_GROUP_TAG_NAME}}"
+
+usage() {
+  cat <<EOF
+Start a bosh-lite instance in AWS.
+
+Usage:
+  $SCRIPT_NAME <start|stop|destroy|info|ssh>
+
+Actions:
+
+  start         Start bosh-lite
+  stop          Halt bosh-lite
+  destroy       Destroy bosh-lite
+  info          Print bosh-lite info
+  ssh           SSH or execute commands on the bosh-lite instance
+
+Requirements:
+ * Exported \$DEPLOY_ENV variable
+ * Exported AWS credentials as \$AWS_ACCESS_KEY_ID and \$AWS_SECRET_ACCESS_KEY
+ * aws_cli installed
+ * vagrant and vagrant-aws plugin
+
```

Comment:
Not necessary if you do https://github.gds/government-paas/aws-account-wide-terraform/pull/67#issuecomment-35388